From 986c9c7dff52465da398cfd9069d6e6442322b28 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 2 Apr 2018 12:52:32 +0100
Subject: [PATCH 38/42] Set auth-none=1 in conf/rserver-dev.conf in-case we
 need to debug

.. also set session-timeout-minutes=10000000 in conf/rsession-dev.conf
for the same reason.
---
 src/cpp/conf/rserver-dev.conf  | 2 +-
 src/cpp/conf/rsession-dev.conf | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/cpp/conf/rserver-dev.conf b/src/cpp/conf/rserver-dev.conf
index 7d8aac750a..9c62daf773 100644
--- a/src/cpp/conf/rserver-dev.conf
+++ b/src/cpp/conf/rserver-dev.conf
@@ -26,7 +26,7 @@ server-data-dir=/tmp/rstudio-server
 auth-validate-users=0
 
 # always authenticate users (defaults to no-auth if not running as root)
-auth-none=0
+auth-none=1
 
 # custom html for login page
 auth-login-page-html=${CMAKE_CURRENT_SOURCE_DIR}/conf/login.html
diff --git a/src/cpp/conf/rsession-dev.conf b/src/cpp/conf/rsession-dev.conf
index d54e2d03c3..35760b725d 100644
--- a/src/cpp/conf/rsession-dev.conf
+++ b/src/cpp/conf/rsession-dev.conf
@@ -15,7 +15,7 @@
 
 
 # timeout frequently for build iteration and to test suspend
-session-timeout-minutes=1
+session-timeout-minutes=10000000
 
 # see errors in the console
 log-stderr=1
-- 
2.28.0.windows.1

